trulens_eval/trulens_eval/tru.py

```diff
@@ -408,8 +418,8 @@ def run_dashboard(
 
         proc = subprocess.Popen(
             [
-                "streamlit", "run", "--server.headless=True", leaderboard_path,
-                "--", "--database-file", self.db.filename
+              "streamlit", "run", "--server.headless=True", leaderboard_path,
+              "--", "--database-url", self.db.engine.url
```

Comment:
Popen expects string instead of URL. Will need to cast to string.

Reply:
Fixed this. Now noticed that the dashboard is not loading, will investigate.
```
File "/home/lariel/miniconda3/envs/py38_trulens/lib/python3.8/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
File "/home/lariel/workspace/trulens-fork/trulens_eval/trulens_eval/Leaderboard.py", line 139, in <module>
    main()
File "/home/lariel/workspace/trulens-fork/trulens_eval/trulens_eval/Leaderboard.py", line 135, in main
    streamlit_app()
File "/home/lariel/workspace/trulens-fork/trulens_eval/trulens_eval/Leaderboard.py", line 49, in streamlit_app
    df, feedback_col_names = lms.get_records_and_feedback([])
File "/home/lariel/workspace/trulens-fork/trulens_eval/trulens_eval/db_v2/utils.py", line 43, in wrapper
    return func(*args, **kwargs)
File "/home/lariel/workspace/trulens-fork/trulens_eval/trulens_eval/db_v2/db.py", line 110, in get_records_and_feedback
    return AppsExtractor().get_df_and_cols(apps)
File "/home/lariel/workspace/trulens-fork/trulens_eval/trulens_eval/db_v2/db.py", line 119, in get_df_and_cols
    df = pd.concat(self.extract_apps(apps))
File "/home/lariel/miniconda3/envs/py38_trulens/lib/python3.8/site-packages/pandas/core/reshape/concat.py", line 372, in concat
    op = _Concatenator(
File "/home/lariel/miniconda3/envs/py38_trulens/lib/python3.8/site-packages/pandas/core/reshape/concat.py", line 429, in __init__
    raise ValueError("No objects to concatenate")
```